tool/speed.cc

```diff
@@ -3053,7 +3053,9 @@ bool Speed(const std::vector<std::string> &args) {
        !SpeedKEM(selected) ||
 #endif
 #if AWSLC_API_VERSION > 31
```

Comment:
The XAES APIs are introduced in https://github.com/aws/aws-lc/blob/368d4999f5608e92108c6d4b821937107ce8e4a9/include/openssl/base.h#L117
so maybe we need to introduce a new `#if AWSLC_API_VERSION > 34` for them.